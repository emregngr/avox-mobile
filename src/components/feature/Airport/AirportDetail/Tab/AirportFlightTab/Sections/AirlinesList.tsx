import React from 'react'

import { AirportSectionRow } from '@/components/feature/Airport/AirportDetail/AirportSectionRow'
import { AirlineRowCard } from '@/components/feature/Airport/AirportDetail/Tab/AirportFlightTab/Cards/AirlineRowCard'

interface AirlinesListProps {
  airlines: string[]
  iconColor: string,
  title: string
}

export const AirlinesList = ({ airlines, iconColor, title }: AirlinesListProps) => (
  <AirportSectionRow title={title}>
    {airlines?.map((airline, index) => (
      <AirlineRowCard airline={airline} iconColor={iconColor} key={index} />
      ))}
  </AirportSectionRow>
  )
