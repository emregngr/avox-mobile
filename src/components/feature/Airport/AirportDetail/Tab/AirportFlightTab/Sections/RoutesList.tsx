import React from 'react'

import { AirportSectionRow } from '@/components/feature/Airport/AirportDetail/AirportSectionRow'
import { RouteRowCard } from '@/components/feature/Airport/AirportDetail/Tab/AirportFlightTab/Cards/RouteRowCard'
import type { AirportRoute } from '@/types/feature/airport'

interface RoutesListProps {
  iconColor: string
  routes: AirportRoute[]
  title: string
}

export const RoutesList = ({ iconColor, routes, title }: RoutesListProps) => (
  <AirportSectionRow title={title}>
    {routes?.map((route, index) => (
      <RouteRowCard
        destinationIata={route.destinationIata}
        frequency={route.frequency}
        iconColor={iconColor}
        key={index}
      />
    ))}
  </AirportSectionRow>
)
