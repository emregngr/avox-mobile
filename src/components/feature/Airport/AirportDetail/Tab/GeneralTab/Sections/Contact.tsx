import { router } from 'expo-router'
import React, { useCallback, useMemo } from 'react'
import { Linking } from 'react-native'

import { AirportRowItem } from '@/components/feature/Airport/AirportDetail/AirportRowItem'
import { AirportSectionRow } from '@/components/feature/Airport/AirportDetail/AirportSectionRow'
import { getLocale } from '@/locales/i18next'
import useLocaleStore from '@/store/locale'
import type { Airport } from '@/types/feature/airport'

interface ContactProps {
  airportInfo: Airport['airportInfo']
}

export const Contact = ({ airportInfo }: ContactProps) => {
  const { selectedLocale } = useLocaleStore()

  const { contactInfo: { email, phone } = {}, website } = airportInfo ?? {}

  const handlePhonePress = useCallback(async () => {
    if (phone) {
      await Linking?.openURL(`tel:${phone}`)
    }
  }, [phone])

  const handleEmailPress = useCallback(async () => {
    if (email) {
      await Linking?.openURL(`mailto:${email}`)
    }
  }, [email])

  const handleWebsitePress = useCallback(() => {
    if (website) {
      const formattedUrl = website?.startsWith('http') ? website : `https://${website}`

      router.navigate({
        params: {
          title: website,
          webViewUrl: formattedUrl,
        },
        pathname: '/web-view-modal',
      })
    }
  }, [website])

  const localeStrings = useMemo(
    () => ({
      contactInformation: getLocale('contactInformation'),
      email: getLocale('email'),
      phone: getLocale('phone'),
      website: getLocale('website'),
    }),
    [selectedLocale],
  )

  return (
    <AirportSectionRow title={localeStrings.contactInformation}>
      <AirportRowItem
        icon="globe"
        label={localeStrings.website}
        onPress={handleWebsitePress}
        value={website}
      />

      <AirportRowItem
        icon="call"
        label={localeStrings.phone}
        onPress={handlePhonePress}
        value={phone}
      />

      <AirportRowItem
        icon="mail"
        label={localeStrings.email}
        onPress={handleEmailPress}
        value={email}
      />
    </AirportSectionRow>
  )
}
