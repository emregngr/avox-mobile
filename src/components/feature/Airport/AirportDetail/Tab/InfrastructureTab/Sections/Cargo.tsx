import React, { useMemo } from 'react'

import { AirportRowItem } from '@/components/feature/Airport/AirportDetail/AirportRowItem'
import { AirportSectionRow } from '@/components/feature/Airport/AirportDetail/AirportSectionRow'
import { getLocale } from '@/locales/i18next'
import useLocaleStore from '@/store/locale'
import type { Airport } from '@/types/feature/airport'
import { formatNumber } from '@/utils/feature/formatNumber'

interface CargoProps {
  cargo: Airport['cargo']
}

export const Cargo = ({ cargo }: CargoProps) => {
  const { selectedLocale } = useLocaleStore()

  const { annualCargoTonnes, coldStorage, dangerousGoods, terminalCapacityTonnes } = cargo ?? {}

  const formattedAnnualCargo = useMemo(() => formatNumber(annualCargoTonnes), [annualCargoTonnes])

  const formattedTerminalCapacity = useMemo(
    () => formatNumber(terminalCapacityTonnes),
    [terminalCapacityTonnes],
  )

  const coldStorageStatus = useMemo(
    () => (coldStorage ? getLocale('existing') : getLocale('none')),
    [coldStorage],
  )

  const dangerousGoodsStatus = useMemo(
    () => (dangerousGoods ? getLocale('yes') : getLocale('no')),
    [dangerousGoods],
  )

  const localeStrings = useMemo(
    () => ({
      annualShipping: getLocale('annualShipping'),
      cargo: getLocale('cargo'),
      coldStorage: getLocale('coldStorage'),
      dangerousGoods: getLocale('dangerousGoods'),
      terminalCapacity: getLocale('terminalCapacity'),
    }),
    [selectedLocale],
  )

  return (
    <AirportSectionRow title={localeStrings.cargo}>
      <AirportRowItem
        icon="cube"
        label={localeStrings.annualShipping}
        value={formattedAnnualCargo}
      />

      <AirportRowItem
        icon="cube-outline"
        label={localeStrings.terminalCapacity}
        value={formattedTerminalCapacity}
      />

      <AirportRowItem icon="snow" label={localeStrings.coldStorage} value={coldStorageStatus} />

      <AirportRowItem
        icon="warning"
        label={localeStrings.dangerousGoods}
        value={dangerousGoodsStatus}
      />
    </AirportSectionRow>
  )
}
