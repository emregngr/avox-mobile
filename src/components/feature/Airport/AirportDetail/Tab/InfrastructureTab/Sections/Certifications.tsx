import { MaterialCommunityIcons } from '@expo/vector-icons'
import React, { useMemo } from 'react'
import { View } from 'react-native'

import { ThemedText } from '@/components/common/ThemedText'
import { AirportSectionRow } from '@/components/feature/Airport/AirportDetail/AirportSectionRow'
import { getLocale } from '@/locales/i18next'
import useLocaleStore from '@/store/locale'
import useThemeStore from '@/store/theme'
import { themeColors } from '@/themes'
import type { AirportType } from '@/types/feature/airport'

interface CertificationsProps {
  safety: AirportType['safety']
}

export const Certifications = ({ safety }: CertificationsProps) => {
  const { selectedLocale } = useLocaleStore()
  const { selectedTheme } = useThemeStore()

  const colors = useMemo(() => themeColors?.[selectedTheme], [selectedTheme])

  const { certifications } = safety ?? {}

  const localeStrings = useMemo(
    () => ({
      certifications: getLocale('certifications'),
    }),
    [selectedLocale],
  )

  return (
    <AirportSectionRow title={localeStrings.certifications}>
      <View className="flex-row items-center py-2">
        <MaterialCommunityIcons color={colors?.onPrimary100} name="certificate-outline" size={20} />
        <ThemedText className="ml-3" color="text-100" type="body2">
          {certifications}
        </ThemedText>
      </View>
    </AirportSectionRow>
  )
}
