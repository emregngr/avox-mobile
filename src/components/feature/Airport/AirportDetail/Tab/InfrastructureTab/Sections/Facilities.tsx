import React, { useMemo } from 'react'

import FireStationIcon from '@/assets/icons/fireStation.svg'
import TowerIcon from '@/assets/icons/tower.svg'
import { AirportRowItem } from '@/components/feature/Airport/AirportDetail/AirportRowItem'
import { AirportSectionRow } from '@/components/feature/Airport/AirportDetail/AirportSectionRow'
import { getLocale } from '@/locales/i18next'
import useLocaleStore from '@/store/locale'
import type { AirportType } from '@/types/feature/airport'
import { formatNumber } from '@/utils/feature/formatNumber'

interface FacilitiesProps {
  facilities: AirportType['facilities']
  infrastructure: AirportType['infrastructure']
}

export const Facilities = ({ facilities, infrastructure }: FacilitiesProps) => {
  const { selectedLocale } = useLocaleStore()

  const { fireCategory, towerHeightM } = infrastructure ?? {}

  const {
    checkinTimeAvg,
    freeWifi,
    googleMapsRating,
    hasMetro,
    loungeCount,
    parkingCapacityVehicles,
    securityQueueTime,
  } = facilities ?? {}

  const formattedParkingCapacity = useMemo(
    () => formatNumber(parkingCapacityVehicles),
    [parkingCapacityVehicles],
  )

  const formattedSecurityTime = useMemo(() => `~${securityQueueTime}`, [securityQueueTime])
  const formattedCheckinTime = useMemo(() => `~${checkinTimeAvg}`, [checkinTimeAvg])

  const formattedGoogleRating = useMemo(
    () => `${googleMapsRating?.toFixed(1)} / 5`,
    [googleMapsRating],
  )

  const metroStatus = useMemo(() => (hasMetro ? getLocale('yes') : getLocale('no')), [hasMetro])
  const wifiStatus = useMemo(() => (freeWifi ? getLocale('free') : getLocale('paid')), [freeWifi])

  const localeStrings = useMemo(
    () => ({
      checkIn: getLocale('checkIn'),
      facilities: getLocale('facilities'),
      fireDepartmentCategory: getLocale('fireDepartmentCategory'),
      googleRating: getLocale('googleRating'),
      lounges: getLocale('lounges'),
      metro: getLocale('metro'),
      parking: getLocale('parking'),
      security: getLocale('security'),
      tower: getLocale('tower'),
      wifiStatus: getLocale('wifiStatus'),
    }),
    [selectedLocale],
  )

  return (
    <AirportSectionRow title={localeStrings.facilities}>
      <AirportRowItem customIcon={TowerIcon} label={localeStrings.tower} value={towerHeightM} />

      <AirportRowItem
        customIcon={FireStationIcon}
        label={localeStrings.fireDepartmentCategory}
        value={fireCategory}
      />

      <AirportRowItem
        icon="car-outline"
        label={localeStrings.parking}
        value={formattedParkingCapacity}
      />

      <AirportRowItem icon="subway" label={localeStrings.metro} value={metroStatus} />

      <AirportRowItem icon="wifi" label={localeStrings.wifiStatus} value={wifiStatus} />

      <AirportRowItem icon="food-fork-drink" label={localeStrings.lounges} value={loungeCount} />

      <AirportRowItem
        icon="timer-outline"
        label={localeStrings.security}
        value={formattedSecurityTime}
      />

      <AirportRowItem
        icon="clock-outline"
        label={localeStrings.checkIn}
        value={formattedCheckinTime}
      />

      <AirportRowItem
        icon="star-outline"
        label={localeStrings.googleRating}
        value={formattedGoogleRating}
      />
    </AirportSectionRow>
  )
}
