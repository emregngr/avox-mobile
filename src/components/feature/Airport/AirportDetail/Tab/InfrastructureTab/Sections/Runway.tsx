import React, { useMemo } from 'react'

import RunwayIcon from '@/assets/icons/runway.svg'
import { AirportRowItem } from '@/components/feature/Airport/AirportDetail/AirportRowItem'
import { AirportSectionRow } from '@/components/feature/Airport/AirportDetail/AirportSectionRow'
import { getLocale } from '@/locales/i18next'
import useLocaleStore from '@/store/locale'
import type { Airport } from '@/types/feature/airport'
import { formatNumber } from '@/utils/feature/formatNumber'

interface RunwayProps {
  infrastructure: Airport['infrastructure']
}

export const Runway = ({ infrastructure }: RunwayProps) => {
  const { selectedLocale } = useLocaleStore()
  const {
    runwayCount,
    runways: { ilsCategory, lengthM, pcn, surface },
  } = infrastructure ?? {}

  const formattedRunwayLength = useMemo(() => formatNumber(lengthM), [lengthM])

  const localeStrings = useMemo(
    () => ({
      ilsCategory: getLocale('ilsCategory'),
      pcn: getLocale('pcn'),
      runway: getLocale('runway'),
      runwaym: getLocale('runwaym'),
      surface: getLocale('surface'),
      trackInformation: getLocale('trackInformation'),
    }),
    [selectedLocale],
  )

  return (
    <AirportSectionRow title={localeStrings.trackInformation}>
      <AirportRowItem customIcon={RunwayIcon} label={localeStrings.runway} value={runwayCount} />

      <AirportRowItem icon="resize" label={localeStrings.runwaym} value={formattedRunwayLength} />

      <AirportRowItem
        icon="layers"
        label={localeStrings.surface}
        value={surface}
      />

      <AirportRowItem icon="code" label={localeStrings.pcn} value={pcn} />

      <AirportRowItem icon="eye" label={localeStrings.ilsCategory} value={ilsCategory} />
    </AirportSectionRow>
  )
}
