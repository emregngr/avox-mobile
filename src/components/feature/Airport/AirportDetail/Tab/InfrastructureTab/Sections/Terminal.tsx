import React, { useMemo } from 'react'

import ApronIcon from '@/assets/icons/apron.svg'
import { AirportRowItem } from '@/components/feature/Airport/AirportDetail/AirportRowItem'
import { AirportSectionRow } from '@/components/feature/Airport/AirportDetail/AirportSectionRow'
import { getLocale } from '@/locales/i18next'
import useLocaleStore from '@/store/locale'
import type { Airport } from '@/types/feature/airport'
import { formatNumber } from '@/utils/feature/formatNumber'

interface TerminalProps {
  infrastructure: Airport['infrastructure']
}

export const Terminal = ({ infrastructure }: TerminalProps) => {
  const { selectedLocale } = useLocaleStore()
  const {
    airportAreaHectares,
    apronCount,
    baggageCapacity,
    passengerCapacity,
    terminalAreaHectares,
    terminalCount,
  } = infrastructure ?? {}

  const localeStrings = useMemo(
    () => ({
      airportArea: getLocale('airportArea'),
      apron: getLocale('apron'),
      baggageCapacity: getLocale('baggageCapacity'),
      luggagePerHour: getLocale('luggage/hour'),
      millionPerYear: getLocale('million/year'),
      numberOfPassengers: getLocale('numberOfPassengers'),
      terminal: getLocale('terminal'),
      terminalArea: getLocale('terminalArea'),
      terminalandCapacity: getLocale('terminalandCapacity'),
    }),
    [selectedLocale],
  )

  const formattedPassengerCapacity = useMemo(
    () => `${passengerCapacity} ${localeStrings.millionPerYear}`,
    [passengerCapacity],
  )

  const formattedBaggageCapacity = useMemo(
    () => `${formatNumber(baggageCapacity)} ${localeStrings.luggagePerHour}`,
    [baggageCapacity],
  )

  const formattedTerminalAreaHectares = useMemo(
    () => formatNumber(terminalAreaHectares),
    [terminalAreaHectares],
  )

  const formattedAirportAreaHectares = useMemo(
    () => formatNumber(airportAreaHectares),
    [airportAreaHectares],
  )

  return (
    <AirportSectionRow title={localeStrings.terminalandCapacity}>
      <AirportRowItem
        icon="people"
        label={localeStrings.numberOfPassengers}
        value={formattedPassengerCapacity}
      />

      <AirportRowItem
        icon="briefcase"
        label={localeStrings.baggageCapacity}
        value={formattedBaggageCapacity}
      />

      <AirportRowItem icon="terminal" label={localeStrings.terminal} value={terminalCount} />

      <AirportRowItem
        icon="grid"
        label={localeStrings.terminalArea}
        value={formattedTerminalAreaHectares}
      />

      <AirportRowItem
        icon="map"
        label={localeStrings.airportArea}
        value={formattedAirportAreaHectares}
      />

      <AirportRowItem customIcon={ApronIcon} label={localeStrings.apron} value={apronCount} />
    </AirportSectionRow>
  )
}
