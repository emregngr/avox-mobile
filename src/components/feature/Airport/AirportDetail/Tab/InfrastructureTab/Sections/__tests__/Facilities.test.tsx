import { render } from '@testing-library/react-native'
import React from 'react'

import { Facilities } from '@/components/feature/Airport/AirportDetail/Tab/InfrastructureTab/Sections/Facilities'
import { getLocale } from '@/locales/i18next'
import useLocaleStore from '@/store/locale'
import { formatNumber } from '@/utils/feature/formatNumber'

jest.mock('@/locales/i18next')

const mockedGetLocale = getLocale as jest.MockedFunction<typeof getLocale>

jest.mock('@/store/locale')

const mockedUseLocaleStore = useLocaleStore as jest.MockedFunction<typeof useLocaleStore>

jest.mock('@/utils/feature/formatNumber')

const mockedFormatNumber = formatNumber as jest.MockedFunction<typeof formatNumber>

const mockedAirportRowItem = jest.fn()
const mockedAirportSectionRow = jest.fn()

jest.mock('@/assets/icons/fireStation.svg', () => 'FireStationIcon')
jest.mock('@/assets/icons/tower.svg', () => 'TowerIcon')

jest.mock('@/components/feature/Airport/AirportDetail/AirportRowItem', () => {
  const { View } = require('react-native')
  return {
    AirportRowItem: (props: any) => {
      mockedAirportRowItem(props)
      return <View testID="mocked-row-item" />
    },
  }
})

jest.mock('@/components/feature/Airport/AirportDetail/AirportSectionRow', () => {
  const { View } = require('react-native')
  return {
    AirportSectionRow: (props: any) => {
      mockedAirportSectionRow(props)
      return <View testID="mocked-section-row">{props.children}</View>
    },
  }
})

const mockedFacilities: any = {
  checkinTimeAvg: 15,
  freeWifi: true,
  googleMapsRating: 4.8,
  hasMetro: true,
  loungeCount: 12,
  parkingCapacityVehicles: 40000,
  securityQueueTime: 10,
}

const mockedInfrastructure: any = {
  fireCategory: 'CAT 10',
  towerHeightM: 90,
}

beforeEach(() => {
  mockedUseLocaleStore.mockReturnValue({ selectedLocale: 'en' })

  mockedGetLocale.mockImplementation((key: string) => key)

  mockedFormatNumber.mockImplementation(num => (num ? `formatted_${num}` : ''))
})

describe('Facilities Component', () => {
  it('should render the section with the correct title', () => {
    render(<Facilities facilities={mockedFacilities} infrastructure={mockedInfrastructure} />)
    expect(mockedGetLocale).toHaveBeenCalledWith('facilities')
    expect(mockedAirportSectionRow).toHaveBeenCalledWith(
      expect.objectContaining({ title: 'facilities' }),
    )
  })

  it('should render nine row items', () => {
    render(<Facilities facilities={mockedFacilities} infrastructure={mockedInfrastructure} />)
    expect(mockedAirportRowItem).toHaveBeenCalledTimes(9)
  })

  it('should call formatNumber for parking capacity', () => {
    render(<Facilities facilities={mockedFacilities} infrastructure={mockedInfrastructure} />)
    expect(mockedFormatNumber).toHaveBeenCalledWith(40000)
  })

  it('should pass correct props to all AirportRowItems', () => {
    render(<Facilities facilities={mockedFacilities} infrastructure={mockedInfrastructure} />)

    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ customIcon: 'TowerIcon', label: 'tower', value: 90 }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({
        customIcon: 'TowerIcon',
        label: 'fireDepartmentCategory',
        value: 'CAT 10',
      }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({
        icon: 'car-outline',
        label: 'parking',
        value: 'formatted_40000',
      }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ icon: 'subway', label: 'metro', value: 'yes' }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ icon: 'wifi', label: 'wifiStatus', value: 'free' }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ icon: 'food-fork-drink', label: 'lounges', value: 12 }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ icon: 'timer-outline', label: 'security', value: '~10' }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ icon: 'clock-outline', label: 'checkIn', value: '~15' }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ icon: 'star-outline', label: 'googleRating', value: '4.8 / 5' }),
    )
  })

  it('should display correct status when booleans are false', () => {
    const falseFacilities = { ...mockedFacilities, hasMetro: false, freeWifi: false }
    render(<Facilities facilities={falseFacilities} infrastructure={mockedInfrastructure} />)

    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ label: 'metro', value: 'no' }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ label: 'wifiStatus', value: 'paid' }),
    )
  })

  it('should handle undefined props gracefully', () => {
    render(<Facilities facilities={undefined as any} infrastructure={undefined as any} />)

    expect(mockedAirportRowItem).toHaveBeenCalledTimes(9)
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ label: 'tower', value: undefined }),
    )
    expect(mockedAirportRowItem).toHaveBeenCalledWith(
      expect.objectContaining({ label: 'parking', value: '' }),
    )
  })
})

describe('Facilities Component Snapshot', () => {
  it('should render the Facilities Component successfully', () => {
    const { toJSON } = render(
      <Facilities facilities={mockedFacilities} infrastructure={mockedInfrastructure} />,
    )

    expect(toJSON()).toMatchSnapshot()
  })
})
