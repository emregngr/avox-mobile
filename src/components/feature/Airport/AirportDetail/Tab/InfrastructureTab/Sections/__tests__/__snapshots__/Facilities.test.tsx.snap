// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Facilities Component Snapshot should render the Facilities Component successfully 1`] = `
<View
  testID="mocked-section-row"
>
  <View
    testID="mocked-row-item"
  />
  <View
    testID="mocked-row-item"
  />
  <View
    testID="mocked-row-item"
  />
  <View
    testID="mocked-row-item"
  />
  <View
    testID="mocked-row-item"
  />
  <View
    testID="mocked-row-item"
  />
  <View
    testID="mocked-row-item"
  />
  <View
    testID="mocked-row-item"
  />
  <View
    testID="mocked-row-item"
  />
</View>
`;
